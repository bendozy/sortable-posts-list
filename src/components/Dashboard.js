import React from 'react';
import PostList from './PostList';
import ActionList from './ActionList';

const Dashboard = () => (
  <div className="Dashboard container flex flex-wrap my-10 px-5">
    <div className="w-full md:w-1/2 px-8">
      <PostList />
    </div>
    <div className="w-full md:w-1/2 px-8">
      <ActionList />
    </div>
  </div>
);

export default Dashboard;
