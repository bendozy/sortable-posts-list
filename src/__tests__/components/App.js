import React from 'react';
import toJson from 'enzyme-to-json';
import { Provider } from 'react-redux';
import App from '../../components/App';
import Dashboard from '../../components/Dashboard';

describe('<App />', () => {
  it('should render a <App /> component', () => {
    const wrapper = shallow(<App />);

    expect(wrapper.find(Provider)).toHaveLength(1);
    expect(wrapper.find(Dashboard)).toHaveLength(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
