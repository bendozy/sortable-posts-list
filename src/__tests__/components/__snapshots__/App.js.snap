// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render a <App /> component 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Dashboard />
</Provider>
`;
